nservicebus/azure-storage-queues/configuration.md

```diff
@@ -6,11 +6,7 @@ tags:
 - Cloud
 ---
 
-The default settings can be overridden by adding a configuration section called `AzureServiceBusQueueConfig` to the web.config or app.config files:
-
-snippet:AzureStorageQueueConfig
-
-The following values can be modified using this configuration setting:
+Azure Storage Queues Transport can be configured using following parameters. Depending on the version, parameters' values can  be configured using a custom configuration section or code-first approach. Please find the description of the parameters below:
```

Comment:
I would keep the first sentence and move the second sentence to after the list. The 3rd sentence is then not needed.